testing/tests/interproc.py

```diff
@@ -51,7 +51,7 @@ def testIUnsanitized(self):
         getFunctionASTsByName("interproc_arg_tainter_test")
         .match{ it.type == "CallExpression" && it.code.startsWith('sink12')}
         .statements()
-        .iUnsanitized(NO_RESTRICTION, { it2 -> if(it2.code.matches('.*source12.*')) [20] else [] } )
+        .iUnsanitized(Object.metaClass.NO_RESTRICTION, { it2 -> if(it2.code.matches('.*source12.*')) [20] else [] } )
```

Comment:
Not a huge fan of this uglier reference, but also unsure if it's possible reference closures in the global context without a full path.